src/cpu/x64/jit_brgemm_conv.cpp

```diff
@@ -1187,18 +1187,26 @@ status_t brgemm_convolution_fwd_t<isa>::init(engine_t *engine) {
         for (int owb = 0; owb < jcp.nb_ow; owb++) {
             const auto ow_b = owb * jcp.ow_block;
             const auto ow_e = nstl::min(ow_b + jcp.ow_block, OW);
+            const auto ow_l = ow_e - ow_b;
 
             if (owb == 0) {
                 comp_owb[owb] = 0;
                 continue;
             }
 
-            for_(int i = 0; i < comp_ow_l; i++)
-            for (int ow = ow_b; ow < ow_e; ow++) {
```

Comment:
Looks like this could be updated to `ow <= ow_e` and `if` condition below could be updated to not do `- 1` and that's it.